vhdbuilder/packer/install-dependencies.sh

```diff
@@ -472,22 +472,26 @@ PATCHED_HYPERKUBE_IMAGES="
 1.15.11-hotfix.20200529.1
 1.15.11-hotfix.20200714.1
 1.15.11-hotfix.20200817.1
+1.15.11-hotfix.20200824.1
 1.15.12-hotfix.20200623.1
 1.15.12-hotfix.20200714.1
 1.15.12-hotfix.20200817.1
+1.15.12-hotfix.20200824.1
 1.16.9-hotfix.20200529.1
 1.16.10-hotfix.20200623.1
 1.16.10-hotfix.20200714.1
 1.16.10-hotfix.20200817.1
+1.16.10-hotfix.20200824.1
 1.16.13-hotfix.20200714.1
 1.16.13-hotfix.20200817.1
+1.16.13-hotfix.20200824.1
 1.17.3-hotfix.20200601.1
 1.17.7-hotfix.20200624
 1.17.7-hotfix.20200714.1
 1.17.7-hotfix.20200714.2
-1.17.7-hotfix.20200817.1
```

Comment:
why do we remove this?

Reply:
https://github.com/Azure/AgentBaker/pull/263#discussion_r478041412

Reply:
thanks @sozercan !

Reply:
#271  to catch future issue